scanapi/tree/request_node.py

```diff
@@ -115,6 +115,10 @@ def retry(self):
         return self.spec.get(RETRY_KEY)
 
     def run(self):
+        """Run request and yours tests.
+        Returns:
+            [dict]: values required to render template.
```

Comment:
How about ` [dict]: HTTP response and test results with request node name, to be used by the report template.`